stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/resiliency/commons.bal

```diff
@@ -0,0 +1,81 @@
+package ballerina.net.http.resiliency;
+
+import ballerina.net.http;
+
+
+//Constants to represent Failover connector actions.
+public const string FORWARD = "forward";
+public const string GET = "get";
+public const string POST = "post";
+public const string DELETE = "delete";
+public const string OPTIONS = "options";
+public const string PUT = "put";
+public const string PATCH = "patch";
+public const string HEAD = "head";
+
+// Integer constants that are used to identify Failover connector actions.
+public const int FORWARD_INT = 0;
+public const int GET_INT = 1;
+public const int POST_INT = 2;
+public const int DELETE_INT = 3;
+public const int OPTIONS_INT = 4;
+public const int PUT_INT = 5;
+public const int PATCH_INT = 6;
+public const int HEAD_INT = 7;
+
+
+// makes the actual endpoints call according to the http operation passed in.
+public function invokeEndpoint (string path, http:OutRequest outRequest, http:InRequest inRequest,
+                         int requestAction, http:HttpClient httpClient)
+(http:InResponse, http:HttpConnectorError) {
+
+    endpoint<http:HttpClient> endPoint {
+    }
+
+    bind httpClient with endPoint;
+
+    http:InResponse inResponse;
+    http:HttpConnectorError httpConnectorError = {};
+
+    if (GET_INT == requestAction) {
+        inResponse, httpConnectorError = endPoint.get(path, outRequest);
```

Comment:
Isn't better if we directly return within each if statement (i.e. `return endPoint.get(path, outRequest); `etc) and add else clause at the end to represent non matching case?